pandas/tests/frame/test_dtypes.py

```diff
@@ -622,3 +623,75 @@ def test_astype_str(self):
                         'NaT                       NaT' in result)
         self.assertTrue('2 2013-01-03 2013-01-03 00:00:00-05:00 '
                         '2013-01-03 00:00:00+01:00' in result)
+
+    def test_values_is_ndarray_with_datetime64tz(self):
+        df = DataFrame({
+            'A': date_range('20130101', periods=3),
+            'B': date_range('20130101', periods=3, tz='US/Eastern'),
+        })
+
+        for col in [
+            ["A"],
+            ["A", "A"],
+            ["A", "B"],
+            ["B", "B"],
+            ["B"],
+        ]:
+            arr = df[col].values
+            fst_elem = arr[0, 0]
+
+            self.assertEqual(type(arr), np.ndarray)
+            if "B" in col:
+                self.assertEqual(type(fst_elem), pd.Timestamp)
+            else:
+                self.assertEqual(type(fst_elem), np.datetime64)
+
+    def test_values_dtypes_with_datetime64tz(self):
```

Comment:
seeing all the self.assertEqual, I though we were using "nosetests" style and avoided to use any pytest features. BTW, in http://pandas.pydata.org/pandas-docs/stable/contributing.html#running-the-test-suite, it is mentionned to run the tests suite with nosetests and not pytest. I could never run the suite with nosetests but well with pytest